confluent/docker_utils/cub.py

```diff
@@ -261,6 +263,68 @@ def check_connect_ready(host, port, service_timeout):
         return False
 
 
+def check_ksql_server_ready(host, port, service_timeout):
+    """Waits for KSQL server to be ready.
+
+    Args:
+        host: Hostname where KSQL server is hosted.
+        port: KSQL server port.
+        timeout: Time in secs to wait for the service to be available.
+
+    Returns:
+        False, if the timeout expires and KSQL server is not ready, True otherwise.
+
+    """
+
+    # Check if you can connect to the endpoint
+    status = wait_for_service(host, port, service_timeout)
+
+    if status:
+        # Check if service is responding as expected to basic request
+        url = "http://%s:%s" % (host, port)
```

Comment:
I think this should be `host:port/info`. `host:port` serves a 307 redirect to it, so shouldn't we just hit the intended endpoint directly?

Reply:
Great suggestion, thanks @rmoff